src/main/k/working/imp/merge/1.merge-golden

```diff
@@ -25,27 +25,20 @@
             VarDotVar0:SortGeneratedCounterCell{}
         )
     ),
-    \and{SortGeneratedTopCell{}}(
-        \top{SortGeneratedTopCell{}}(),
-        Lbl'-LT-'generatedTop'-GT-'{}(
-            Lbl'-LT-'T'-GT-'{}(
-                Lbl'-LT-'k'-GT-'{}(
-                    kseq{}(VarI:SortKItem{}, VarDotVar2:SortK{})
-                ),
-                Lbl'-LT-'state'-GT-'{}(
-                    /* builtin: */
-                    Lbl'Unds'Map'Unds'{}(
-                        /* element: */ Lbl'UndsPipe'-'-GT-Unds'{}(
-                            /* Inj: */ inj{SortId{}, SortKItem{}}(
-                                VarX:SortId{}
-                            ),
-                            VarI:SortKItem{}
-                        ),
-                        /* opaque child: */ VarDotVar3:SortMap{}
-                    )
+    Lbl'-LT-'generatedTop'-GT-'{}(
```

Comment:
Why has the output from these two tests changed?

Reply:
Because it had to be changed previously when I added exists for the first time. now it reverted to its initial form.